cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/operationrunner/InMemorySourceControlOperationRunner.java

```diff
@@ -62,13 +71,14 @@ public class InMemorySourceControlOperationRunner implements SourceControlOperat
   public PushAppResponse push(NamespaceId namespace,
                               RepositoryConfig repoConfig,
                               ApplicationDetail appToPush,
-                              CommitMeta commitDetails) throws PushFailureException, NoChangesToPushException,
```

Comment:
Should we add back the `SourceControlException` in throw list?

Reply:
Not needed as SourceControlException is unchecked